bootstrap/py_bootstrap.bat

```diff
@@ -0,0 +1,15 @@
+@echo off
+
+set SCRIPT_DIR=%~dp0
+set VENV_DIR=%SCRIPT_DIR%venv
+
+if exist %VENV_DIR% (
+    call %VENV_DIR%\Scripts\activate
+) else (
+    echo Creating virtualenv
+    python -m venv %VENV_DIR%
+    call %VENV_DIR%\Scripts\activate
+    pip install -r %SCRIPT_DIR%requirements.txt
+)
+
+python %SCRIPT_DIR%main.py
```

Comment:
I like to use quotes, even when they're not strictly required. Windows cmd scripts have a strange syntax, so you usually need to quote the whole argument, not just the part containing the variable reference, otherwise the quotes just become part of the argument.
```suggestion
set "SCRIPT_DIR=%~dp0"
set "VENV_DIR=%SCRIPT_DIR%venv"

if exist "%VENV_DIR%" (
    call "%VENV_DIR%\Scripts\activate"
) else (
    echo Creating virtualenv
    python -m venv "%VENV_DIR%"
    call "%VENV_DIR%\Scripts\activate"
    pip install -r "%SCRIPT_DIR%requirements.txt"
)

python "%SCRIPT_DIR%main.py"
```